pkg/kustomize/kustomize.go

```diff
@@ -70,12 +70,16 @@ func componentImageFor(component v1.ComponentKind) types.Image {
 	}
 	image := os.Getenv(envVarFor[component])
 	if image != "" {
-		if len(strings.Split(image, "@")) != 2 {
-			panic(envVarFor[component] + " must use manifest digest reference")
+		if len(strings.Split(image, "@")) == 2 {
+			imageOverride.NewName = strings.Split(image, "@")[0]
+			imageOverride.Digest = strings.Split(image, "@")[1]
+		} else if len(strings.Split(image, ":")) == 2 {
+			imageOverride.NewName = strings.Split(image, ":")[0]
+			imageOverride.NewTag = strings.Split(image, ":")[1]
+		} else {
```

Comment:
We shouldn't default to `latest`. Just panic here and say a tag or manifest digest must be specified.